src/jwt.ts

```diff
@@ -1,12 +1,12 @@
 import * as express from "express";
-import jwt from "express-jwt";
+import {expressjwt as jwt} from "express-jwt";
 
 export default function jwtHandler(credentialsRequired: boolean) {
   return [
-    jwt({secret: process.env.APP_KEY!, credentialsRequired}),
+    jwt({secret: process.env.APP_KEY!, credentialsRequired, algorithms: ['HS256']}),
```

Comment:
dziwne, ze obecnie na produkcji dziala to bez tego parametru :/ przypuszczalnie masz racje co to HS256

Reply:
@adam-boduch Spodziewam się że TypeScript wymusza obecność parametru, ale w runtimie JavaScript robi tam jakiś default. Możliwe że obraz był zbudowany bez sprawdzania typów 🤔 

Nie wiem czy `HS256` jest odpowiedni, taki pojawiał się w dokumentacji: https://www.npmjs.com/package/express-jwt/v/8.2.0#usage